src/osd/OSD.cc

```diff
@@ -160,6 +161,8 @@
 #undef dout_prefix
 #define dout_prefix _prefix(_dout, whoami, get_osdmap_epoch())
 
+using namespace std::chrono;
```

Comment:
Updated as following. furthermore, do we need to convert other `ceph_now` to `ceph::coarse_mono_clock`? I am happy to do this if needed.

```diff
diff --git a/src/osd/OSD.cc b/src/osd/OSD.cc
index b333b4e..c8f3485 100644
--- a/src/osd/OSD.cc
+++ b/src/osd/OSD.cc
@@ -161,7 +161,6 @@
 #undef dout_prefix
 #define dout_prefix _prefix(_dout, whoami, get_osdmap_epoch())
 
-using namespace std::chrono;
 
 const double OSD::OSD_TICK_INTERVAL = 1.0;
 
@@ -2213,7 +2212,7 @@ bool OSD::asok_command(string admin_command, cmdmap_t& cmdmap, string format,
     auto start = ceph::coarse_mono_clock::now();
     store->compact();
     auto end = ceph::coarse_mono_clock::now();
-    duration<double> time_span = duration_cast<duration<double>>(end - start);
+    chrono::duration<double> time_span = chrono::duration_cast<chrono::duration<double>>(end - start);
     dout(1) << "finished manual compaction in " << time_span.count() << " seconds" << dendl;
     f->open_object_section("compact_result");
     f->dump_float("elapsed_time", time_span.count());
@@ -6631,7 +6630,7 @@ void OSD::do_command(Connection *con, ceph_tid_t tid, vector<string>& cmd, buffe
     auto start = ceph::coarse_mono_clock::now();
     store->compact();
     auto end = ceph::coarse_mono_clock::now();
-    duration<double> time_span = duration_cast<duration<double>>(end - start);
+    chrono::duration<double> time_span = chrono::duration_cast<chrono::duration<double>>(end - start);
     dout(1) << "finished manual compaction in " << time_span.count() << " seconds" << dendl;
     ss << "compacted omap in " << time_span.count() << " seconds";
   }
```